interpreter/activation.go

```diff
@@ -48,8 +48,14 @@ func EmptyActivation() Activation {
 //
 // The input `bindings` may either be of type `Activation` or `map[string]interface{}`.
 //
-// When the bindings are a `map` form whose values are not of `ref.Val` type, the values will be
-// converted to CEL values (if possible) using the `types.DefaultTypeAdapter`.
+// Lazy bindings may be supplied in either of the following forms:
+// - func() interface{}
```

Comment:
I'd prefer not to use a type alias here since I find `native` misleading. The value can be either CEL's representation, a native value, or a lazy supplier that produces a CEL or native result.

Reply:
The issue is that `interface{}` means that here, but it might mean something else in another location. E.g. I don't think every consumer of `interface{}` can handle a lazy supplier. But whether in code or commments, this is for the future, so resolving for now.